analyzere/base_resources.py

```diff
@@ -318,6 +318,12 @@ def download_yelt(self, auto_retry=True, **params):
         return request_raw(
             'get', path, params=params, auto_retry=auto_retry).content
 
+    def back_allocation(self, source_id, auto_retry=True, **params):
+        params['source_id'] = source_id
+        path = '%s/back_allocations' % self._get_path(self.id)
+        data = request('get', path, auto_retry=auto_retry, params=params)
+        return analyzere.LayerView.retrieve(data['layer_view']['ref_id'])
```

Comment:
I'd just return `data`. When the client accesses `data['layer_view']` it should resolve it automatically for them I believe.